client/components/site-selector/add-site.jsx

```diff
@@ -15,10 +15,8 @@ import PopoverMenu from 'components/popover/menu';
 import PopoverMenuItem from 'components/popover/menu-item';
 import { recordTracksEvent } from 'state/analytics/actions';
 import config from 'config';
-import sitesFactory from 'lib/sites-list';
 import { abtest } from 'lib/abtest';
-
-const sites = sitesFactory();
+import { hasJetpackSites } from 'state/selectors';
```

Comment:
It would be nice to use this new selector in the `sidebar` as well - currently we're manually doing a `getSites( state ).some( s => s.jetpack )` right now. Could be done in another PR though.